include/deal.II/base/bounding_box.h

```diff
@@ -77,15 +77,25 @@ class BoundingBox
   const std::pair<Point<spacedim,Number>,Point<spacedim,Number>> &get_boundary_points () const;
 
   /**
+   * Merging function
```

Comment:
Please extend this comment so say something like "grow the current object to also contain @p other"